app/Kconfig

```
@@ -248,6 +248,39 @@ config ZMK_RGB_UNDERGLOW_ON_START
 #ZMK_RGB_UNDERGLOW
 endif
 
+menuconfig ZMK_BACKLIGHT
+	bool "LED backlight"
+	select PWM
+	select LED
+    select LED_PWM
+
+if ZMK_BACKLIGHT
+
+config ZMK_BACKLIGHT_LEVELS
+	int "Number of brightness levels"
+    range 2 100
+	default 6
+
+config ZMK_BACKLIGHT_LEVEL_START
+	int "Default brightness level"
+    range 1 ZMK_BACKLIGHT_LEVELS
+	default 3
```

```suggestion
	default ZMK_BACKLIGHT_LEVELS
```
Since 2 is a valid amount of backlight levels, 3 would be out of range, right?

---

It should automatic change if you use `west build -t menuconfig`, but that's a good idea.